views/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```

Comment:
Shouldn't this be index.ejs? Index.ejs is what is called in index.js.

Reply:
ah yeah, i was running that locally so i changed some of the files youre right
if its running in replit, it should be index,ejs
i'll config it so that it is supposed to run in replit in my next PR